packages/plugin-menu/src/manager.ts

```diff
@@ -50,9 +50,9 @@ export class Manager {
                 if (config.disabled) {
                     const disabled = config.disabled(view);
                     if (disabled) {
-                        config.$.classList.add('disabled');
+                        config.$.setAttribute('disabled', 'true');
```

Comment:
Setting the disabled attribute of the button means if someone wants to leave the buttons in place when they're disabled to stop the menu jumping around and changing, that's now possible to do.